pyoptsparse/pySNOPT/source/f2py/snopt.pyf

```diff
@@ -396,50 +396,50 @@ python module snopt ! in
             double precision dimension(lenrw) :: rw
             integer optional,check(len(rw)>=lenrw),depend(rw) :: lenrw=len(rw)
         end subroutine snoptc
-        subroutine snkerc(start,m,n,ne,nname,nncon,nnobj,nnjac,iobj,objadd,prob,userfg,snlog,snlog2,sqlog,snstop,jcol,indj,locj,bl,bu,names,hs,x,pi,rc,inform,mincw,miniw,minrw,ns,ninf,sinf,obj,cu,lencu,iu,leniu,ru,lenru,cw,lencw,iw,leniw,rw,lenrw) ! in :snopt:snoptc.f
+        subroutine snkerc(start,m,n,nej,nnames,nncon,nnobju,nnjac,iobju,objuadd,prob,userfg,snlog,snlog2,sqlog,snstop,jcol,indj,locj,bl,bu,names,hs,x,pi,rc,info,mincw,miniw,minrw,ns,ninf,sinf,obj,cu,lencu,iu,leniu,ru,lenru,cw,lencw,iw,leniw,rw,lenrw) ! in :snopt7_python:snopt7_python.f90
```

Comment:
Is this supposed to duplicate the official SNOPT python wrapper? I searched for `snkerc` [here](https://github.com/snopt/snopt-python/blob/master/solvers/f2py/snopt7_python.f90) but could not find it.

Reply:
Not really, they don't support `snkerc` but call `snoptc` which wraps it. We use `snkerc` because it allows us to pass in a user-specified SNSTOP function (and log functions but we don't do that).

This change is cosmetic and merely mirrors the variable names within the SNOPT source code to make things consistent.

Reply:
I see. What does this comment mean though `! in :snopt7_python:snopt7_python.f90` ?